contracts/minter-suite/Minters/MinterMerkle/MinterMerkleV1.sol

```diff
@@ -68,8 +78,10 @@ contract MinterMerkleV1 is ReentrancyGuard, IFilteredMinterMerkleV0 {
 
     /// projectId => merkle root
     mapping(uint256 => bytes32) public projectMerkleRoot;
-    /// projectId => purchaser address => has purchased one or more mints
-    mapping(uint256 => mapping(address => bool)) public projectMintedBy;
+
+    /// projectId => purchaser address => qty of mints purchased for project
+    mapping(uint256 => mapping(address => uint256))
+        public projectUserInvocations;
```

Comment:
Nit: consider naming this: projectUserMintInvocations

Reply:
updated!